components/layout/Header.vue

```diff
@@ -1013,920 +533,5 @@ export default {
-
-            li {
-                flex-direction: column-reverse;
-                gap: $spacer;
-                align-items: center;
-
-                a {
-                    width: 100%;
-                }
-            }
-        }
-    }
-
-    :deep(.github .btn) {
-        font-weight: bold !important;
-        font-size: 12px !important;
-    }
-
-        &:not(.transparent) {
-            :deep(.github) {
-                background: #333336;
-
-                &:hover {
-                    .btn-dark {
-                        background: #18181B !important;
-                    }
-
-                    .btn-outline-dark {
-                        background: #333336 !important;
-                    }
-                }
-            }
-            :deep(.github .btn-dark) {
-                color: $white-6;
-            }
-
-            :deep(.github .btn-outline-dark) {
-                background: #18181B;
-                color: $white-6;
-            }
-        }
-
-    .btn:hover {
-        color: $purple-36;
-    }
-
-    &.transparent {
-        background: transparent;
-        box-shadow: none;
-
-        a,
-        a.nav-link,
-        button.navbar-toggler,
-        &.btn.search {
-            color: var(--bs-white);
-        }
-
-
-        :deep(.github .btn-outline-dark) {
-            color: var(--bs-white);
-        }
-
-        .navbar-collapse {
-            ul.navbar-nav {
-                li {
-                    a.nav-link {
-
-                        &.show,
-                        &:hover {
-                            color: $secondary;
-                            background: rgba($gray-100, 5%);
-                        }
-                    }
-                }
-            }
-        }
-
-        @include media-breakpoint-down(lg) {
-            &.open {
-                background: #200149;
-            }
-
-            .navbar-collapse ul.navbar-nav li .dropdown-menu .dropdown-item {
-                color: var(--bs-white);
-                --bs-dropdown-link-hover-bg: #{rgba($gray-100, 5%)};
-                --bs-dropdown-link-active-bg: #{rgba($gray-100, 5%)};
-
-                &:hover {
-
-                    .material-design-icon,
-                    span {
-                        color: var(--bs-white);
-                    }
-                }
-
-                .material-design-icon,
-                span {
-                    color: var(--bs-white);
-                }
-
-                p {
-                    color: var(--bs-gray-400);
-
-                    span {
-                        color: var(--bs-white);
-                    }
-                }
-            }
-
-            .dropdown-menu {
-                background: transparent;
-            }
-        }
-    }
-}
-</style>
-
-<style lang="scss">
-@import "../../assets/styles/variable";
-
-.wrapper.announce {
-    nav {
-        top: 3rem;
-    }
-}
-</style>
+@import "../../assets/styles/components/header.scss";
+</style>
```

Comment:
NIT: You can use the vue structure here. Is it useful... maybe not.
`<style lang="scss" src="../../assets/styles/components/header.scss" scoped>`